spring-kafka/src/main/java/org/springframework/kafka/core/StreamsBuilderFactoryBean.java

```diff
@@ -36,14 +36,15 @@
  *
  * @author Artem Bilan
  * @author Ivan Ursul
+ * @author Soby Chacko
  *
  * @since 1.1.4
  */
 public class StreamsBuilderFactoryBean extends AbstractFactoryBean<StreamsBuilder> implements SmartLifecycle {
 
 	private static final int DEFAULT_CLOSE_TIMEOUT = 10;
 
-	private final StreamsConfig streamsConfig;
+	private StreamsConfig streamsConfig;
 
 	private final CleanupConfig cleanupConfig;
```

Comment:
Nit-pick: the fields order should be like:
`static final`
`static`
`final`
non-`volatile`
`volatile`

So, since the `streamsConfig` isn't `final` any more should be moved below `cleanupConfig`.